iamy.go

```diff
@@ -44,6 +44,7 @@ func main() {
 		skipCfnTagged   = kingpin.Flag("skip-cfn-tagged", fmt.Sprintf("Shorthand for --skip-tagged %s", cloudformationStackNameTag)).Bool()
 		skipTagged      = kingpin.Flag("skip-tagged", "Skips IAM entities (or buckets associated with bucket policies) tagged with a given tag").Strings()
 		includeTagged   = kingpin.Flag("include-tagged", "Includes IAM entities (or buckets associated with bucket policies) tagged with a given tag").Strings()
+		skipPaths       = kingpin.Flag("skip-path", fmt.Sprintf("Skips IAM entities that have a path starting with the supplied prefixes, repeat flag for multiple prefixes")).Strings()
```

Comment:
If we want to take this approach (specifically handling prefixes), I'd suggest we try to align with a convention of some kind. I notice `aws iam list-groups` takes a `--path-prefix` rather than `--path`, so perhaps we could use `--skip-path-prefix`?

On the other hand, I'm wondering if we're being unnecessarily specific by adding a flag specifically for excluding prefixes — could we take a more general approach, something like `--exclude '/aws-reserved/*'`?

Reply:
Very much prefer the `-prefix` addition to the argument name over `--exclude`, way too much ambiguity in that for my comfort.

Reply:
Pushed a change to reflect this a3548ed